agent/tests/test_run_test_pipeline.py

```diff
@@ -0,0 +1,11 @@
+from agent import cli
+
+
+def test_run_pipeline(cli_runner):
+    result = cli_runner.invoke(cli.run_test_pipeline, catch_exceptions=False)
+    print(f"run_test_pipeline: {result.exit_code} {result.output}")
+
+    assert result.exit_code == 0
+    assert "Source run_test_pipeline created" in result.output
+    assert "Pipeline run_test_pipeline is starting" in result.output
+    assert "Test run OK" in result.output
```

Comment:
I recommend not checking for a specific text in the output, you'll need to update tests every time you change output text and if you forget you'll know about it only when the test runs on github, that takes about 10 minutes